dev-python/dbuspython/dbuspython-1.2.14.recipe

```diff
@@ -0,0 +1,112 @@
+SUMMARY="The original dbus implementation for Python"
+DESCRIPTION="dbus-python is the original Python binding \
+for dbus, the reference implementation of the D-Bus protocol."
+HOMEPAGE="https://www.freedesktop.org/wiki/Software/DBusBindings/"
+COPYRIGHT="2002-2010 Red Hat Inc.
+	2003 David Zeuthen
+	2002-2003 CodeFactory AB
+	2003 James Willcox
+	2004 Rob Taylor
+	2004 Anders Carlsson
+	2005 Colin Walters
+	2006 Steve Frécinaux
+	2009-2011 Nokia Corporation
+	2010-2012 Mike Gorse
+	2016 Simon McVittie
+	2005-2019 Collabora Ltd."
+LICENSE="MIT
+	AFL v2.1
+	GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://dbus.freedesktop.org/releases/dbus-python/dbus-python-$portVersion.tar.gz"
+CHECKSUM_SHA256="b10206ba3dd641e4e46411ab91471c88e0eec1749860e4285193ee68df84ac31"
+SOURCE_DIR="dbus-python-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	dbuspython = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libdbus_1
+	devel:libglib_2.0
+	"
+
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:bash
+	cmd:gawk
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtool
+	cmd:make
+	cmd:pkg_config
+	"
+
+PYTHON_PACKAGES=(python python36 python3)
+PYTHON_VERSIONS=(2.7 3.6 3.7)
```

Comment:
If we want to have this for secondary architecture you should remove python (2.7), or add a check for targetArchitecture to ditch python2.7 for x86_gcc2 (secondary architecture) and leave that for x86_64